scala/src/extension.ts

```diff
@@ -79,4 +79,68 @@ export function activate(context: ExtensionContext) {
     ]
   })
 
+  // sbt command support
+  let terminal: Terminal = null;
+  const runCommandInIntegratedTerminal = (args: string[], cwd: string) => {
+    if (terminal === null) {
+      terminal = window.createTerminal('sbt');
+    }
+    terminal.show();
+    terminal.sendText(args.join(' '));
+  }
+
+  const runSbtCommand = (args: string[], cwd?: string) => {
+    workspace.saveAll().then(() => {
+      if (!cwd) {
+        cwd = workspace.rootPath;
+      }
+      args.splice(0, 0, 'sbt');
```

Comment:
Would it be possible to run `sbt` just once, and pipe all the other commands in the running sbt instance? This would save some time, since sbt is slow to start.